site/docs/guides/contributor/release-process/egeria-charts-release.md

```diff
@@ -0,0 +1,29 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the Egeria project 2020. -->
+
+### Egeria charts Release process
+
+#### Obtaining releases / artifacts for Egeria charts 
+
```

Comment:
The egeria charts release process is lightweight. Branches are not used. Instead we simply continually increase the version (including support for prerelease versions that are hidden).


Since the charts aggregate many components, as well as themselves being under constant development, the release cycle will typically be more often that egeria itself.